builder/gen-dockerfile/contents/src/detect-setup.ts

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
I'm a little confused by this.

This code doesn't seem to be part of the build pipeline; it's part of fingerprinting, which is going to remain client-side for the time being